certs/g_cacert.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Oups, my bad. I created this file when I was retrieving public certificates to make the B2Access test instance SSL work. Nothing confidential, but it should be removed. (Reminder to myself: sending PR past 10PM is a bad idea)